src/SpecDataHolder.t.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.6;
+
+import "forge-std/Test.sol";
+import { IERC165 } from "./IERC165.sol";
+
+import { IERC721Metadata } from "./IERC721Metadata.sol";
+import { IERC4973 } from "ERC4973/interfaces/IERC4973.sol";
+import { Badges } from "./Badges.sol";
+import { SpecDataHolder } from "./SpecDataHolder.sol";
+import { Raft } from "./Raft.sol";
+
+contract SpecDataHolderTest is Test {
+  Raft raft;
+  Badges badges;
+  SpecDataHolder specDataHolder;
+
+  address passiveAddress = 0x0f6A79A579658E401E0B81c6dde1F2cd51d97176;
```

Comment:
Could we move all ofthese into environment variables and read from there?
https://book.getfoundry.sh/cheatcodes/external